specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,55 @@
+# SKU Price Details
+
+The **SKU Price Details** column represents a list of relevant properties shared by all charges with the same [**SKU Price ID**](#skupriceid). These properties provide qualitative and quantitative details about the service represented by a **SKU Price ID**. This can enable practitioners to calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. These properties can also help a practitioner understand the specifics of a **SKU Price ID** and differentiate it other **SKU Price IDs**.
+
+
+The *SkuPriceDetails* column adheres to the following requirements:
+
+- The *SkuPriceDetails* column MUST be in [*KeyValueFormat*](#key-valueformat).
+- The key for a property SHOULD be formatted in [PascalCase](#glossary:pascalcase).
+- The properties (both keys and values) contained in the *SkuPriceDetails* column MUST be shared across all charges having the same *SkuPriceId*, subject to the below provisions.
+  - Additional properties (key-value pairs) MAY be added to *SkuPriceDetails* going forward for a given *SkuPriceId*.
+  - Properties SHOULD NOT be removed from *SkuPriceDetails* for a given *SkuPriceId*, once they have been included.
+  - Individual properties (key-value pairs) SHOULD NOT be modified for a given *SkuPriceId* and SHOULD remain consistent over time.
+- The key for a property SHOULD remain consistent across comparable SKUs having that property and the values for this key SHOULD remain in a consistent format.
+- The *SkuPriceDetails* column MUST NOT contain properties which are not applicable to the corresponding *SkuPriceId*.
+- The *SkuPriceDetails* column MAY contain properties which are already captured in other dedicated columns.
+- If a property has a numeric value, it MUST represent the value for a single [*PricingUnit*](#pricingunit).
+- The *SkuPriceDetils* column MUST be present in the billing data when the provider includes a *SkuPriceID*.
```

Comment:
```suggestion
- The *SkuPriceDetails* column MUST be present in the billing data when the provider includes a *SkuPriceId*.
```

Question: 
Should we also specify when it MUST and MUST NOT be null? For instance, _SkuPriceDetails MUST be null when SkuPriceId is null, and MUST NOT be null when SkuPriceId is not null._